src/app/root-store/server-data/user-tags/user-tags.selectors.ts

```diff
@@ -98,3 +98,8 @@ export const getNavigationOrder = createSelector(
   getUserTagsEntities,
   (entities): string => entities[userTagKeys.navigationOrder].value
 );
+
+export const getKeyboardLayout = createSelector(
+  getUserTagsEntities,
+  (entities): string => entities[userTagKeys.keyboardLayoutForVms] && entities[userTagKeys.keyboardLayoutForVms].value
```

Comment:
All tags should exist see examples of other tags. Add this tag to default-config and SystemTagsService

Reply:
![screenshot from 2018-08-22 14-05-28](https://user-images.githubusercontent.com/18456914/44449642-dce19f80-a618-11e8-8d1e-74148f06740a.png)